src/Map/assets/test/leaflet_controller.test.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import { Application, Controller } from '@hotwired/stimulus';
+import { getByTestId, waitFor } from '@testing-library/dom';
+import { clearDOM, mountDOM } from '@symfony/stimulus-testing';
+import LeafletController from '../src/leaflet_controller';
+
+// Controller used to check the actual controller was properly booted
+class CheckController extends Controller {
```

Comment:
to discuss: should those tests be in their bridge directories ?

Reply:
Yep they will